controllers/schools.js

```diff
@@ -49,11 +49,12 @@ const createBucket = (req, res, next) => {
         });
 };
 
-const getStorageProviders = () => {
-    return [
-        {label: 'AWS S3', value: 'awsS3'}
-    ];
-};
+const getStorageProviders = res => [
+	{
+		label: res.locals.theme.short_title,
```

Comment:
seems good idea, this has been changed in server 2 months ago too

Reply:
As I mentioned in the PR description: "Now containing the changes made in PR #48 which was never reviewed." - this was the AWS S3 label changed that was merged in client/server but not in SHD and is now included in this PR.